src/Common/tests/System.Net/Sockets/Performance/SocketPerformanceTests.cs

```diff
@@ -55,13 +65,24 @@ public SocketPerformanceTests(ITestOutputHelper log)
                 "Test execution is expected to be shorter than " + expectedMilliseconds + " but was " + milliseconds);
         }
 
+        public void ClientServerTest(
+            SocketImplementationType serverType, 
+            SocketImplementationType clientType, 
+            int iterations, 
+            int bufferSize, 
+            int socketInstances, 
+            long expectedMilliseconds)
+        {
+            ClientServerTest(0, serverType, clientType, iterations, bufferSize, socketInstances, expectedMilliseconds);
```

Comment:
Nit: I was a bit confused on what 0 meant here (since this is a _Client_ / Server test and client port 0 doesn't make sense). It would be nice to add a comment to explain that 0 means automatic port assignment.